lib/rouge/lexers/python.rb

```diff
@@ -215,14 +216,9 @@ def current_string
           | x[a-fA-F0-9]{2}
           | [0-7]{1,3}
           )
-        )x do
-          if current_string.type? "r"
-            token Str
-          else
-            token Str::Escape
-          end
-          pop!
-        end
+        )x, Str::Escape, :pop!
+
+        rule %r/./, Error, :pop!
```

Comment:
@tuxu Should this be an error? I was looking at what happens if I write `"\p"` in the Python REPL and it looks like it comes back without any issues. Should `"\p"` just be highlighted as a plain string?

Reply:
After thinking more about this, I think it's wrong to mark these as errors. Non-valid escape sequences should just be tokenised as regular strings, I think. The new commit does that.